README.md

```diff
@@ -21,6 +22,7 @@ Out of the box, it has tested support for select [DNS providers](#dns-providers)
 
 1. Copy the contents of this repo to your device at `/data/udm-le`.
 2. Edit `/data/udm-le/udm-le.env` and tweak variables to meet your needs.
+   - On UniFi UNAS Pro, change DNS Resolver to: `DNS_RESOLVER="127.0.0.53:53"`
```

Comment:
I think maybe a better spot is a comment in the .env file near the DNS_RESOLVER that mentions if you're having issues on a UNAS device, etc.